### PR TITLE
Fix 404 on Helm chart icon

### PR DESCRIPTION
**Pull request by undera**
The old icon URL seem to be dead. Favicon is large enough to serve the purpose.

### Description of your changes

Just a cosmetic change in chart metadata.

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: cluster/charts/crossplane/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: 0.0.1
 description: Crossplane is an open source Kubernetes add-on that enables platform teams to assemble infrastructure from multiple vendors, and expose higher level self-service APIs for application teams to consume.
 name: crossplane
 version: 0.0.1
-icon: https://crossplane.io/images/favicon_192x192.png
+icon: https://crossplane.io/favicon.ico
```
[the remark this answers is not in the document]
Comment (undera) [reply cluster/charts/crossplane/Chart.yaml]: That's even better. I've changed the URL to that

**Review by hasheddan — APPROVED** [review]
Thanks @undera! Would you mind squashing into a single commit here?

**Comment by undera** [comment]
> Thanks @undera! Would you mind squashing into a single commit here?

Done

**Comment by undera** [comment]
I guess CI failure is unrelated to my changes?

**Review by hasheddan — COMMENTED** [review]
@undera yep -- should be fixed with https://github.com/crossplane/crossplane/pull/3589 if you want to rebase 👍🏻

**Comment by undera** [comment]
I did rebase, but it somehow takes other two commits into this PR. I never used to rebase my branches in the past, so I'm not sure if this is how it should work.

**Review by hasheddan — COMMENTED** [review]
@undera I went ahead and fixed it up here -- let me know if anything looks out of order to you!

**Comment by undera** [comment]
I'm completely blank on what's going on and how you managed to modify my branch. But I'll take that anyway :-)

**Pull request merged**